modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx

```diff
@@ -78,12 +78,13 @@ export default function AttributeFields({
 					<ClayInput
 						aria-describedby={`${nameId}fieldFeedback`}
 						aria-required={true}
+						data-testid="name"
```

Comment:
testid's should be unique but this will be the same for every attribute row. It should be namespaced with an index like nameId and others.